tests/unit_test/linux/logging-stack/logging_stack.h

```diff
@@ -1,182 +0,0 @@
-/*
- * AWS IoT Device SDK for Embedded C V202009.00
```

Comment:
Why has this file been removed? It is purposely added to support logging for unit tests...